mobile/src/libs/components/header/header.tsx

```diff
@@ -20,13 +20,15 @@ type Properties = {
   isArrowVisible?: boolean;
   badgeCount?: number;
   isProfileVisible?: boolean;
+  isChat?: boolean;
```

Comment:
Why did you only change the sizes for the chat page?

Reply:
@danilittaa because my task was only about it

Reply:
Perhaps the QA team simply doesn't realize that the same component is used everywhere. Why not make these changes in the header once so that we don't have to fix it on every page later?😅

Reply:
@danilittaa  i guess that change is need to be done only here 🤔 because its chat name (not page named chat with all chats) and this name can be different every time, it can be quite long etc. and if we speak about our headers for screens like meditation or notes , they have appropriate size and they look good. we only can have problems with chat header for now